apps/site/assets/css/_trip-plan-results.scss

```diff
@@ -0,0 +1,75 @@
+.m-trip-plan-results__itinerary-container {
+  margin-bottom: $base-spacing * 1.5;
+
+  .c-expandable-block__header {
+    font-size: $base-spacing;
+    line-height: 1.5;
```

Comment:
does this need some sort of unit or relations to `$base-spacing`?

Reply:
line-height is OK without units, [MDN on unitless line-height](
https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#Prefer_unitless_numbers_for_line-height_values). the grid buttons are relatively new and set a line-height this way. I think this is OK since line-height being unitless has similar reasoning as using $base-spacing.